### PR TITLE
Update project specification documentation

### PR DESCRIPTION
**Pull request by MartinWheelerMT**
Update project specification document to include additional information around terminologies
Added appendix of all available HA Ciphers
Reformatted and fixed errors in README.md

This is a non-breaking change affecting documentation only.

Have also regenerated the HTML version of the updated specification yaml file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -373,12 +373,12 @@ A `mesh.sh` bash script exists for testing or debugging MESH. For more informati
...
 
-A mock implementation of the MESH API is available for local development. The latest version is in Github at
-[mattd-kainos/fake-mesh](https://github.com/jamespic/fake-mesh). _It is a fork of [jamespic/fake-mesh](https://github.com/jamespic/fake-mesh)._
+A mock implementation of the MESH API is available for local development. The latest version is in GitHub at
+[mattd-kainos/fake-mesh](https://github.com/jamespic/fake-mesh). _It is a fork of [jamespic/fake-mesh](https://github.com/jamespic/fake-mesh).
 
 The [nhsdev Docker Hub](https://hub.docker.com/repository/docker/nhsdev/fake-mesh) hosts released fake-mesh images.
 
-### Common Issues
+### Common Issuesx
```
Comment (adrianclay) [inline README.md]: Issuesx

File: specification/appendix/appendix-1-ha-ciphers.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+## Appendix 1 - List of HA Ciphers
+
+Below is a list of all available HA Ciphers.  This list is definitive and will not change before NHAIS is retired
+
+This is sourced from the
+[NHAIS Developer Document Library](https://digital.nhs.uk/services/nhais/nhais-developer-document-library).
```
Comment (adrianclay) [inline specification/appendix/appendix-1-ha-ciphers.md:6]: Do you know which of the resources in that list? I wouldn't know where to begin.

File: README.md [hunk 3]
```diff
@@ -373,12 +373,12 @@ A `mesh.sh` bash script exists for testing or debugging MESH. For more informati
 
 #### Fake MESH
 
-A mock implementation of the MESH API is available for local development. The latest version is in Github at
-[mattd-kainos/fake-mesh](https://github.com/jamespic/fake-mesh). _It is a fork of [jamespic/fake-mesh](https://github.com/jamespic/fake-mesh)._
+A mock implementation of the MESH API is available for local development. The latest version is in GitHub at
+[mattd-kainos/fake-mesh](https://github.com/jamespic/fake-mesh). _It is a fork of [jamespic/fake-mesh](https://github.com/jamespic/fake-mesh).
```
Comment (adrianclay) [inline README.md]: Minor: This `mattd-kainos` link looks wrong.

File: specification/nhais-adaptor.yaml [hunk 4]
```diff
@@ -85,21 +93,40 @@ info:
...
+      > This is not the same the as the practice ODS code, and will be provided with by PCSE at the point that the link 
+      is established. This is a Local Code of PCSE’s choosing.
 
       #### GP Trading Partner Code
 
       The GP Trading Partner Code is a four-character (uppercase letters and digits) code used in the interchange header
       of the EDIFACT messages. This may also be referred to as the "GP Link Code".
-
+    
+      > For English Practices, the supplier will be provided with this by PCSE (Primary Care Support England, Capita),
```
Comment (adrianclay) [inline specification/nhais-adaptor.yaml:105]: Not sure about calling out Capita here.

File: README.md (hunk 1 above)
Comment (MartinWheelerMT) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (MartinWheelerMT) [reply README.md]: have found the correct link and updated

File: specification/appendix/appendix-1-ha-ciphers.md (hunk 2 above)
Comment (MartinWheelerMT) [reply specification/appendix/appendix-1-ha-ciphers.md:6]: added additional link to actual appendix on that page

File: specification/nhais-adaptor.yaml (hunk 4 above)
Comment (MartinWheelerMT) [reply specification/nhais-adaptor.yaml:105]: removed Capita. I had included as the original email sent with the details was corrected from `PCSE (Primary Care Support England)` - to PCSE (Primary Care Support England, Capita) and the PCSE website is very explicit on it being a capita service!

**Review by adrianclay — APPROVED** [review]

**Pull request merged**